### PR TITLE
Scrutinizer patch 1

### PR DESCRIPTION
**Pull request by jaapio**
Set copyright to correct format
Some style issues detected by Scrutinizer

### CODE REVIEW

File: src/phpDocumentor/Reflection/Php/Factory/DocBlock.php [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
- * This file is part of phpDocumentor.
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- *
- * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
- * @license   http://www.opensource.org/licenses/mit-license.php MIT
- * @link      http://phpdoc.org
- */
+     * This file is part of phpDocumentor.
```
Comment (mvriel) [inline src/phpDocumentor/Reflection/Php/Factory/DocBlock.php]: This indentation is too much

File: src/phpDocumentor/Reflection/Php/Factory/Function_.php [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
- * This file is part of phpDocumentor.
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- *
- * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
- * @license   http://www.opensource.org/licenses/mit-license.php MIT
- * @link      http://phpdoc.org
- */
+     * This file is part of phpDocumentor.
```
Comment (mvriel) [inline src/phpDocumentor/Reflection/Php/Factory/Function_.php]: This indentation is too much

File: src/phpDocumentor/Reflection/Php/Factory/Method.php [hunk 3]
```diff
@@ -1,19 +1,18 @@
...
- * This file is part of phpDocumentor.
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- *
- * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
- * @license   http://www.opensource.org/licenses/mit-license.php MIT
- * @link      http://phpdoc.org
- */
+     * This file is part of phpDocumentor.
```
Comment (mvriel) [inline src/phpDocumentor/Reflection/Php/Factory/Method.php]: This indentation is too much

File: src/phpDocumentor/Reflection/Php/Factory/Property.php [hunk 4]
```diff
@@ -1,19 +1,18 @@
...
- * This file is part of phpDocumentor.
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- *
- * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
- * @license   http://www.opensource.org/licenses/mit-license.php MIT
- * @link      http://phpdoc.org
- */
+     * This file is part of phpDocumentor.
```
Comment (mvriel) [inline src/phpDocumentor/Reflection/Php/Factory/Property.php]: This indentation is too much

File: src/phpDocumentor/Reflection/Php/Factory/PropertyIterator.php [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
- * This file is part of phpDocumentor.
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- *
- * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
- * @license   http://www.opensource.org/licenses/mit-license.php MIT
- * @link      http://phpdoc.org
- */
+     * This file is part of phpDocumentor.
```
Comment (mvriel) [inline src/phpDocumentor/Reflection/Php/Factory/PropertyIterator.php]: This indentation is too much

**Comment by mvriel** [comment]
Looks good, I am going to merge this; can you fix those 5 incidents of too much indentation in a next PR?

**Comment by jaapio** [comment]
I will, strange that this happens, since those patches are made by scrutinizer

**Pull request merged**